runtime/lua/vim/lsp/buf.lua

```diff
@@ -64,20 +64,32 @@ function M.completion(context)
   return request('textDocument/completion', params)
 end
 
-function M.formatting(options)
+local function formatting_params(options)
```

Comment:
Isn't it better to move to `util.make_formatting_params` like `util.make_position_params`.

Reply:
Sure, why not :) The possibility did occur to me when I saw `util.make_position_params`, I just wasn't sure what the policy on adding functions to the `util` submodule was.

Reply:
The policy is basically, if it's useful, we can expose them here in util. We'll let util grow as needed and if something has maybe 10ish related functions, we'll consider a new module or file. Not a hard and fast rule yet as we're still feeling out all the little details on how we're going to handle lua dev in core.